Documentation/CreateRemoveVMs.rst

```diff
@@ -35,7 +36,9 @@ Create VM
 
     * A network can be specified multiple times to create multiple adapters connected to the same network.
 
-
+  * **--storage-type** - Storage backing type.

```

Comment:
This, in theory should only be required if there is more than 1 node in the cluster and DRBD has been enabled... maybe worth updating #75